### ISSUE TITLE
Escaping backquotes

### ISSUE THREAD

**Issue by dlip** [mattwparas/steel#101]
Is escaping backquotes done this way, or is it still unimplemented?

```
54 │       [else '(you can\'t go that way.)])))
   │                      ^ Parse: Unexpected character: '\\'
```

**Comment by mattwparas** [comment]
Good catch, looks like this isn't handled correctly. I'm currently re-implementing the tokenizer which should make handling this a bit easier. I'll follow up with the PR to replace that when its ready, and link to some test cases to handle it

**Comment by mattwparas** [comment]
This should get fixed in #102
